agent/agent-tooling/gradle/dependency-locks/runtimeClasspath.lockfile

```diff
@@ -87,14 +97,29 @@ net.bytebuddy:byte-buddy-agent:1.10.18
 net.bytebuddy:byte-buddy:1.10.18
 net.java.dev.jna:jna-platform:5.7.0
 net.java.dev.jna:jna:5.7.0
+net.minidev:accessors-smart:1.2
+net.minidev:json-smart:2.3
 org.apache.commons:commons-lang3:3.11
 org.apache.commons:commons-text:1.9
 org.apache.httpcomponents:httpclient:4.5.13
 org.apache.httpcomponents:httpcore:4.4.13
 org.checkerframework:checker-qual:3.12.0
 org.codehaus.woodstox:stax2-api:4.2.1
 org.jetbrains.kotlin:kotlin-bom:1.4.21
+org.jetbrains:annotations:15.0
+org.linguafranca.pwdb:KeePassJava2-dom:2.1.4
```

Comment:
curious to know: do we need all these? should we exclude some of them if not used at all?